### ISSUE TITLE
Compatibility with docker-machine

### ISSUE THREAD

**Issue by iamluc** [iamluc/docker-hostmanager#15]
I suggest to deprecate environment variable `DOCKER_ENTRYPOINT`, and replace in code

```
$client = new DockerClient([], $input->getOption('entrypoint'));
```

by

```
 $client = DockerClient::createWithEnv();
```

It should add `docker-machine` compatibility

**Comment by iamluc** [comment]
Suggested code does not work with PHP 5.6: https://github.com/stage1/docker-php/issues/105

**Comment by iamluc** [comment]
Fixed by #16
